packages/core/helper-plugin/lib/src/utils/setHexOpacity.js

```diff
@@ -0,0 +1,6 @@
+const setHexOpacity = (hex, alpha) =>
```

Comment:
Can you add a small story for this so we have some kind of documentation?

You can put it in the utils section.